### PR TITLE
SUL23-135 | @jdwjdwjdw | Add global typography styles

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- [SUL23-135](https://stanfordits.atlassian.net/browse/SUL23-135): FE-DEV | style Typography for SUL theme
- Add global styles for headings, update tailwind config
- Manually add section header styles in relevant files
- Update link styles where relevant
- Update caption text where relevant

# Review By (Date)
- When convenient

# Steps to Test
1. Checkout branch or view preview at https://deploy-preview-60--stanford-library.netlify.app/sws-test-314-typography
2. Build out elements noted in [the ticket](https://stanfordits.atlassian.net/browse/SUL23-135), and confirm they look correct, or view [preview](https://deploy-preview-60--stanford-library.netlify.app/sws-test-314-typography)
3. Review code 🌴 

# Associated Issues and/or People
- [SUL23-135](https://stanfordits.atlassian.net/browse/SUL23-135): FE-DEV | style Typography for SUL theme

[SUL23-135]: https://stanfordits.atlassian.net/browse/SUL23-135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/(public)/tailwind/plugins/components/typography/sul-typography.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      const components = {
+        'h1, h2, h3, h4, h5': {
+          fontFamily: '"Source Serif Pro", Georgia, Times, "Times New Roman", serif',
+        },
+        'h2': {
+          letterSpacing: '-0.005em',
+        },
+        'h3': {
+          letterSpacing: '-0.003em',
+        },
```
Comment (yvonnetangsu) [inline app/(public)/tailwind/plugins/components/typography/sul-typography.js:15]: @jdwjdwjdw I think these styles belong more in a base.js file not as TW components 😄

Comment (yvonnetangsu) [reply app/(public)/tailwind/plugins/components/typography/sul-typography.js:15]: For example, on MyAccount, we add a project base.js file which includes some link style overrides
https://github.com/SU-SWS/adapt-my-account/blob/dev/src/tailwind/plugins/base/base.js

File: tailwind.config.js [hunk 2]
```diff
@@ -12,6 +12,11 @@ for (i = 1; i <= 4; i++) {
   threeColumn[`1-1-${i}`] = `1fr 1fr ${i}fr`;
 }
 
+const path = require('path');
+
+// Path to custom Tailwind plugins for SUL
+const dir = path.resolve(__dirname, 'app/(public)/tailwind/plugins');
```
Comment (pookmish) [inline tailwind.config.js:18]: Let's put these things into the `app/(public)/styles` directory

**Review by pookmish — CHANGES_REQUESTED** [review]
Just looking to move the files, then this looks pretty good :+1:

**Review by pookmish — APPROVED** [review]

**Pull request merged**